### ISSUE TITLE
Some Suggestions

### ISSUE THREAD

**Issue by Gaoyifei1011** [dongle-the-gadget/WinverUWP#36]
1.目前项目的框架使用的是Winui2.7，未来有机会升级到Winui2.8或Windows App SDK 1.1.2吗？
1. Currently, the framework of the project uses Winui2.7. Is there any chance to upgrade to Winui2.8 or Windows App SDK 1.1.2 in the future?
2.上次我提交的语言列表信息是/Strings/zh-cn，希望您能够将这个文件夹重命名为/Strings/zh-Hans（适用于所有简体中文地区），谢谢。
2. The language list information I submitted last time was /Strings/zh-cn, please rename this folder to /Strings/zh-hans (applicable to all simplified Chinese regions), thank you.
3. 希望下一次发布的时候打捆绑包（X86，X64）
3. Expect bundles for the next release (X86, X64)

**Comment by dongle-the-gadget** [comment]
1. Not sure, since WinUI 2.8 requires 17763+, whereas this app targets 15063+.
2. I'll do it.
3. Currently, it's impossible for me to target all four architectures (x86, x64, ARM, ARM64) in a bundle since the ARM64 has a different target version to the rest of them.

**Comment by Gaoyifei1011** [comment]
> 
1.那这个版本未来就有可能面临抉择了，是选择更高的Winui版本，还是支持低版本系统了。
1. This version may face a choice in the future, whether to choose a higher version of Winui, or support a lower version of the system.
2.这个打捆绑包不是可以不选择所有的架构吗，去除掉ARM，ARM64也不是可以为X86，X64生成捆绑包的吗？
2. This bundle can not select all architecture, remove ARM, ARM64 can not generate bundles for X86, X64?
![image](https://user-images.githubusercontent.com/49179966/179881064-39a80005-a4fa-49da-8c3c-a49aacda9d3b.png)
该安装包仅供我个人使用，不做任何其他用途。
This installation package is for my personal use only, not for any other purpose.

**Comment by dongle-the-gadget** [comment]
Bundling exclusively 2 architectures seem to be quite messy for me.

**Comment by Gaoyifei1011** [comment]
> Bundling exclusively 2 architectures seem to be quite messy for me.

额。。。。。。
Er...

**Comment by Gaoyifei1011** [comment]
> 1. Not sure, since WinUI 2.8 requires 17763+, whereas this app targets 15063+.
> 2. I'll do it.
> 3. Currently, it's impossible for me to target all four architectures (x86, x64, ARM, ARM64) in a bundle since the ARM64 has a different target version to the rest of them.



> 1. Not sure, since WinUI 2.8 requires 17763+, whereas this app targets 15063+.
> 2. I'll do it.
> 3. Currently, it's impossible for me to target all four architectures (x86, x64, ARM, ARM64) in a bundle since the ARM64 has a different target version to the rest of them.

关于第一个我忘记说了，也许您可以在商店上传的时候保留一个适用于旧版本系统的安装包。就像Microsoft Store一样，虽然目前都是基于22XXX的版本，但是商店里面还是保留了11XXX的版本。而且根据市场占有率Win10 1809的版本基本上很少了，15063也是LTSC版本的居多吧。基本上大部分软件都是基于17763+的版本了。这只是我个人的意见，仅供参考。
On the first one, I forgot to mention that maybe you can keep an installation package for older systems when you upload it to the store. Just like the Microsoft Store, although it is currently based on the 22XXX version, the 11XXX version remains in the Store.And according to the market share of Win10 1809 version is basically very few, 15063 is also the majority of LTSC version.
Basically most of the software is based on the 17763+ version. This is just my personal opinion for reference only.
![image](https://user-images.githubusercontent.com/49179966/179903862-b94a974a-12e2-4839-b40e-70f79b985f74.png)
![image](https://user-images.githubusercontent.com/49179966/179903928-2eb8006e-9ba2-4c79-a1df-8a8aeef1a02e.png)
